dist/tools/buildsystem_sanity_check/check.sh

```diff
@@ -87,7 +86,7 @@ check_providing_features_only_makefile_features() {
     patterns+=(-e '^[ ]*FEATURES_PROVIDED *+= *')
 
     pathspec+=("*Makefile\.*")
-
+    pathspec+=(':!dist/tools/m4a-generator/m4agen/templates/board/Makefile.features.j2')
     pathspec+=(":!*Makefile.features")
```

Comment:
These changes allows to the dist/tools build_system_sanity ignore the m4agen templates, that are throwing an error related with FEATURES_PROVIDED , CPU and CPU_MODEL params implemented in the templates/boards/makefiles with extension *.j2